static/app/views/replays/detail/network/index.tsx

```diff
@@ -144,6 +157,14 @@ function NetworkList({replayRecord, networkSpans}: Props) {
         <Item {...columnHandlers}>
           <Text>{network.op.replace('resource.', '')}</Text>
         </Item>
+        <Item {...columnHandlers} numeric>
+          {network.data.size ? (
+            `${network.data.size} B`
+          ) : (
```

Comment:
We have a component or function that formats this nicely.. it should be on storybook

Reply:
Fixed, updated the description with the new capture